### PR TITLE
Created BrowserScreen.js for website embedding 

### PR DESCRIPTION
**Pull request by akummi**
I've created a new component called BrowserScreen.js which allows for easy embedding of websites into the app. This component will be particularly useful in the future when we want to display articles in the education forum section.

I've also added this new component to the navigation tab for easy access. Additionally, I've already used the BrowserScreen.js component to embed our class website for senior design into the education section.

Overall, these changes will greatly improve the user experience by allowing easy access to external content within the app, and laying the foundation for future enhancements to the education section.

Please review and merge at your convenience.

Thank you!

### CODE REVIEW

**Comment by SnellJ2** [comment]
Looks good, I only wonder if this works on both iOS and android.

Otherwise, good work!

File: Carbon/navigation/screens/Main/MainContainer.js [hunk 1]
```diff
@@ -105,7 +107,11 @@ const ForumStack = ({ navigation }) => {
             <Stack.Screen
                 name={' '}
                 //component={ForumScreen}
-                component={QuizScreen}
+
+                component = {BrowserScreen}
+
+                //component={QuizScreen}
```
Comment (SnellJ2) [inline Carbon/navigation/screens/Main/MainContainer.js:113]: Will the quiz screen still be implemented here?

**Review by SnellJ2 — APPROVED** [review]
Overall good

**Comment by erikrramsey** [comment]
Getting an issue running the app due to ScreenNames.js on line 9

**Review by erikrramsey — APPROVED** [review]
Fixed the bug in ScreenNames, looks good now -- the embedded content works perfectly.

**Pull request merged**